DFC.Integration.AVFeed/DFC.Intergration.AVfeed.Function.ClearRecycleBinTests/DeleteRecycleBinTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using DFC.Integration.AVFeed.Data.Interfaces;
+using DFC.Integration.AVFeed.Function.ClearRecycleBin;
+using FakeItEasy;
+using Xunit;
+
+namespace DFC.Integration.AVfeed.Function.ClearRecycleBinTests
+{
+    public class ClearRecycleBinTests
+    {
+        [Fact]
+        public async void ClearRecycleBinAsyncTest()
+        {
+            //Setup
+            var fakeCustomApiContextService = A.Fake<ICustomApiContextService>();
+            var fakeTokenClient = A.Fake<ITokenClient>();
+            var fakeApplicationLogger = A.Fake<IApplicationLogger>();
+            var fakeAuditService = A.Fake<IAuditService>();
+         
+            var clearRecycleBin = new ClearRecycleBin(fakeApplicationLogger, fakeAuditService, fakeCustomApiContextService);
+
+            //Act
+            await clearRecycleBin.ClearRecycleBinAsync();
+
+            //Asserts
+            A.CallTo(() => fakeCustomApiContextService.ClearAVsRecycleBinAsync(A<int>._)).MustHaveHappened();
```

Comment:
Are we not asserting audit service?
Do we need the tokenclient variable?

Reply:
That method only calls one method in its body so that is all we can check for, other then the logging.

Reply:
Please review this as 

             await customApiContextService.ClearAVsRecycleBinAsync(numberToClear);

          ***   await auditService.AuditAsync($"Deleted upto {numberToClear} vacancies from the recycle bin");

 Please also remove redundant token client variable